app/views/users/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
- [ ] You are missing the commented out forms made with the HTML form tag and the one made with the Rails form_tag

I know usually you should not commit commented out code for review but this project explicitly asks you to leave the commented out forms here, make sure to have them and that they both are functional when not commented.

Reply:
Previous iterations of the form using pure HTML and the #form_tag helper have now been re-inserted, checked, and commented out.